packages/stores/src/account/amino-converters.ts

```diff
@@ -24,6 +24,18 @@ const osmosisAminoConverters: Record<
       ],
       aminoType: "osmosis/cl-create-pool",
     },
+  "/osmosis.valsetpref.v1beta1.MsgDelegateToValidatorSet": {
+    ...originalOsmosisAminoConverters[
+      "/osmosis.valsetpref.v1beta1.MsgDelegateToValidatorSet"
+    ],
+    aminoType: "osmosis/MsgDelegateToValidatorSet",
+  },
+  "/osmosis.valsetpref.v1beta1.MsgUndelegateFromValidatorSet": {
+    ...originalOsmosisAminoConverters[
+      "/osmosis.valsetpref.v1beta1.MsgUndelegateFromValidatorSet"
+    ],
+    aminoType: "osmosis/MsgUndelegateFromValidatorSet",
+  },
```

Comment:
Let's update the osmosis commit has to Sishir's changes so we don't have to maintain these exceptions

Reply:
Have rerun telescope twice and the route in the message is still wrong without the amino-converter

Reply:
https://github.com/osmosis-labs/osmosis/blob/1e3149f05a08c5af8d0df9b0392bb554dfc9bc78/proto/osmosis/valset-pref/v1beta1/tx.proto#L117

The Osmosis commit hasn't been updated to include Sishir's changes, which explains the same results